packages/shared/components/molecules/activity-info/SmartContractActivityInformation.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="typescript">
+    import { KeyValueBox } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import { NftActivity } from '@core/wallet'
+    import { parseLayer2Metadata } from '@core/layer-2'
+
+    export let activity: NftActivity
+
+    $: layer2Metadata = parseLayer2Metadata(activity.metadata)
+    $: detailsList = createDetailsList(layer2Metadata)
+
+    function createDetailsList(metadata: typeof layer2Metadata): { [key: string]: { data: unknown } } {
+        return {
+            ...(metadata?.targetContract && {
+                targetContract: { data: metadata.targetContract },
+            }),
+            ...(metadata?.contractFunction && {
+                contractFunction: { data: metadata.contractFunction },
+            }),
+            ...(metadata?.gasBudget && {
+                gasBudget: { data: metadata.gasBudget },
+            }),
```

Comment:
i think the ethereum address would also be relevant here

Reply:
I disagree. Ethereum address should already show up in the top-level destination. So there is no need to duplicate this within smart contract details.

Reply:
thats true, i was just thinking until we're at that state, we can add it here. Currently the contract address is still shown, right?

Reply:
It's already the Ethereum address for the send confirmation popup, but it's not shown in the activity information. This has to be addressed in this PR